docs/upgrade.html

```diff
@@ -27,6 +27,15 @@ <h5><a id="upgrade_350_notable" href="#upgrade_350_notable">Notable changes in 3
         See <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-889%3A+Versioned+State+Stores">KIP-889</a>
         for more details.
     </li>
+    <li>MirrorMaker now uses incrementalAlterConfigs API to synchronize topic configurations instead of the deprecated alterConfigs API.
```

Comment:
MirrorMaker now uses incrementalAlterConfigs API [by default] to synchronize topic configurations instead of the deprecated alterConfigs API.